dashboard/modules/job/sdk.py

```diff
@@ -27,10 +27,17 @@ class ClusterInfo:
     address: str
     cookies: Optional[Dict[str, Any]]
     metadata: Optional[Dict[str, Any]]
+    headers: Optional[Dict[str, Any]]
```

Comment:
Let's also give all additional fields a default value, so this still works if some modules are returning an older version of `ClusterInfo`.